storybook/stories/navigation/Sidebar.stories.js

```diff
@@ -59,11 +59,11 @@ let emptyMenus;
 let currentTheme = sessionStorage.getItem('themeName') || 'light-blue';
 currentTheme = currentTheme.toLowerCase();
 
-storiesOf('Sidebar', module)
+storiesOf('Navigation|Sidebar', module)
```

Comment:
Maybe we don't need `Navigation` main item anymore since we have the navigations working on their places?
Otherwise, if we're keeping it, it's worth then adding Settings Navigation.